manifests/kustomize/cluster/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+- ../base/cache-deployer/cluster
```

Comment:
Also add `crds` folder as a base? so we can skip the step of applying crds in kustomize/README.md and test script to set up kfp standalone.

Note, with current setup, user always need to add an extra step of applying cluster-scoped resources. Therefore, it's more convenient if the extra apply crd instruction turns to first apply all cluster level resources.